nix/haskell.nix

```diff
@@ -149,9 +149,9 @@ CHaP: haskell-nix: nixpkgs-recent: nodePkgs: haskell-nix.cabalProject' [
           haskellPackages.weeder
           haskellPackages.stylish-haskell
 
-          (haskell-nix.tool "ghc963" "haskell-language-server" ({pkgs, ...}: rec {
+          (haskell-nix.tool "ghc964" "haskell-language-server" ({pkgs, ...}: rec {
             # Use the github source of HLS that is tested with haskell.nix CI
-            src = pkgs.haskell-nix.sources."hls-2.5";
+            src = pkgs.haskell-nix.sources."hls-2.6";
```

Comment:
What if you simply don't bump HLS, i.e. leave it at gh963 and hls-2.5?

This will bring both GHC 9.6.4 and GHC 9.6.3 into the nix store. Not sure if that's worth it.

HLS wants to be a special snowflake, that's why the dependency information is different from the other tools. 😅

Reply:
patch applied, https://github.com/input-output-hk/hydra/commit/0b8776ac0bc3f62c8e454feda33327e7d39ff631

seems working

Reply:
@paolino nice!